learnify/mobile/lib/features/search/view/components/search_result_widget.dart

```diff
@@ -0,0 +1,40 @@
+part of '../search_screen.dart';
+
+class SearchResultWidget extends StatefulWidget {
```

Comment:
similarly, I don't think this needs to be Stateful since you are using selector helper for real time management instead of States